tests/openapi/test_strictmode.py

```diff
@@ -461,3 +462,144 @@ def search_request():
+    basic_collection_setup(collection_name=collection_name)  # Clear collection to not depend on other tests
+
+    def upsert_points(ids: list[int]):
+        length = len(ids)
+        payloads = [{} for _ in range(length)]
+        vectors = [[1, 2, 3, 5] for _ in range(length)]
+        return request_with_validation(
+            api='/collections/{collection_name}/points',
+            method="PUT",
+            path_params={'collection_name': collection_name},
+            body={
+                "batch": {
+                    "ids": ids,
+                    "payloads": payloads,
+                    "vectors": vectors
+                }
+            }
+        )
+
+    # Overwriting the same points to trigger cache refreshing
+    for _ in range(33):
+        upsert_points([1, 2, 3, 4, 5]).raise_for_status()
+
+    set_strict_mode(collection_name, {
+        "enabled": True,
+        "max_collection_vector_size_bytes": 240,
+    })
+
+    for _ in range(33):
+        upsert_points([6, 7, 8, 9, 10]).raise_for_status()
+
+    # Max limit has been reached and one of the next requests must fail. Due to cache it might not be the first call!
+    for _ in range(33):
+        failed_upsert = upsert_points([12, 13, 14, 15, 16])
+        if failed_upsert.ok:
+            continue
+        assert "Max vector storage size" in failed_upsert.json()['status']['error']
+        assert not failed_upsert.ok
+        return
+
+    assert False, "Upserting should have failed but didn't"
+
+
+def test_strict_mode_max_collection_size_upsert_batch(collection_name):
+    basic_collection_setup(collection_name=collection_name)  # Clear collection to not depend on other tests
+
+    def upsert_points(ids: list[int]):
+        length = len(ids)
+        payloads = [{} for _ in range(length)]
+        vectors = [[1, 2, 3, 5] for _ in range(length)]
+        return request_with_validation(
+            api='/collections/{collection_name}/points/batch',
+            method="POST",
+            path_params={'collection_name': collection_name},
+            body={
+                "operations": [
+                    {
+                        "upsert": {
+                            "batch": {
+                                "ids": ids,
+                                "payloads": payloads,
+                                "vectors": vectors
+                            }
+                        }
+                    }
+                ]
+            }
+        )
+
+    for _ in range(33):
+        upsert_points([1, 2, 3, 4, 5]).raise_for_status()
+
+    set_strict_mode(collection_name, {
+        "enabled": True,
+        "max_collection_vector_size_bytes": 240,
+    })
+
+    for _ in range(33):
+        upsert_points([6, 7, 8, 9, 10]).raise_for_status()
+
+    # Max limit has been reached and one of the next requests must fail. Due to cache it might not be the first call!
+    for _ in range(33):
```

Comment:
Shouldn't 32 be enough? (also in other places)

```suggestion
    for _ in range(32):
```